.gitignore

```diff
@@ -24,3 +24,4 @@ MatchZoo.egg-info/
 notebooks/wikiqa/.ipynb_checkpoints/*
 .cache
 .tmpdir
+sandbox.py
```

Comment:
Don't ignore your personal files. Just don't add them. Or, add them in your global ignore.

Reply:
OK. I will fix this part and do a force push after cleaning up my files.

Reply:
Fixed.